### ISSUE TITLE
Travis build is failing

### ISSUE THREAD

**Issue by Kreijstal** [fossasia/open-event-server#4846]
**Describe the bug**

Since the last merge, travis has been failing, it has to do with the SQL being offline or being unable to connect with the database, it'd be a nice idea to add exception handling to such cases, and maybe figure out how to make it work again.

**Comment by Kreijstal** [comment]
![image](https://user-images.githubusercontent.com/2415206/40899033-00a30f68-67c5-11e8-88ad-ff40184fd934.png)
Even old merges are failing now.

**Comment by Kreijstal** [comment]
There are two errors the first one is 
```
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 346, in connection
    return self.__connection
AttributeError: 'Connection' object has no attribute '_Connection__connection'
```
and the second one is
```
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 429, in _revalidate_connection
    raise exc.ResourceClosedError("This Connection is closed")
sqlalchemy.exc.ResourceClosedError: This Connection is closed
```

**Comment by mayank8318** [comment]
@Kreijstal As pointed out by @niranjan94 @iamareebjamal https://github.com/travis-ci/travis-ci/issues/8049 this is something we could try?

**Comment by bhaveshAn** [comment]
Looking into it !!

**Comment by Kreijstal** [comment]
This was already fixed by #4854
